ansible/README.md

```diff
@@ -213,12 +213,21 @@ containers:
 Where each item corresponds to a container to be set up and run on the host.
 
 Each `name` should exist as a node in Jenkins and the corresponding `secret`
-should be given. The `os` determines the `Dockerfile` to use to build the host.
-The templates for these can be found in `roles/docker/templates/`.
+should be given.  The `os` determines the `Dockerfile` to use to build the
+host.  The templates for these can be found in `roles/docker/templates/`. 
+
+For testing purposes, the stanza above can be added into
+`host_vars/<docker_host>`, then the master record should be in the
+`inventory.yml` in the secrets repository.
 
 Note that the Docker host itself doesn't need to be known by Jenkins, just the
 containers that are managed there.
 
+The docker-host playbook sets up services which are managed via `systemctl`. 
+If you log into the host sytsem you can view them with `systemctl | grep
+jenkins-` and start and stop individual ones using `systemctl start` and
+`systemctl stop` commands as and when required.
```

Comment:
We could also link to https://github.com/nodejs/build/blob/master/doc/jenkins-guide.md#fixing-machines-with-docker which is the general troubleshooting/how to guide.

Reply:
Done :-)